frontend/public/components/channel-operators/detail-config.jsx

```diff
@@ -45,7 +45,7 @@ export class DetailConfig extends SafetyFirst {
         displayText = this.props.displayFunction(displayText);
       }
       const outdatedClass = this.state.outdated ? 'text-muted': null;
-      return this.props.modal ? <a onClick={this._openModal} className={classNames('co-m-modal-link', outdatedClass)}>{displayText}</a> : <span>{displayText}</span>;
+      return this.props.modal ? <button type="button" onClick={this._openModal} className={classNames('btn btn-link co-modal-btn-link', outdatedClass)}>{displayText}</button> : <span>{displayText}</span>;
```

Comment:
`outdatedClass` is `.text-muted` and it has no effect on a `.btn-link`.  Needs a new CSS rule (ideally a modifier to `co-modal-btn-link`) to change the `.btn-link` text color (note the hover color is dark blue no matter what).

Reply:
Upon inspection this page and the page that calls it `/components/channel-operators/app-version.jsx` look to be orphaned. cc' @spadgett

Reply:
Drop this change since the file appears to be orphaned?

Reply:
Thanks, I opened #1408 to remove the orphaned components